CMakeLists.txt

```diff
@@ -1,22 +1,16 @@
-cmake_minimum_required(VERSION 2.6.4)
+cmake_minimum_required(VERSION 2.8.8)
 
 if (POLICY CMP0048)
   cmake_policy(SET CMP0048 NEW)
 endif (POLICY CMP0048)
 
-project( googletest-distribution )
+project(googletest-distribution)
+set(GOOGLETEST_VERSION 1.9.0)
```

Comment:
Where did 1.9.0 come from?

Reply:
I see these - 
"googletest/CMakeLists.txt:  project(gtest VERSION 1.9.0 LANGUAGES CXX C)
googlemock/CMakeLists.txt:  project(gmock VERSION 1.9.0 LANGUAGES CXX C)
"
Since you are replacing the hard-coded version with the variable, shouldnt the variable be used everywhere?

Reply:
If it isn't being used everywhere, then I made a mistake!

Reply:
I see only this one:

```
$ git grep -nIF '1.9.0'
CMakeLists.txt:8:set(GOOGLETEST_VERSION 1.9.0)
```

Reply:
I still see 2 of them
googletest/CMakeLists.txt:  project(gtest VERSION 1.9.0 LANGUAGES CXX C) - https://github.com/google/googletest/blob/b88511ef640c046ec9c7553b383a2aba64c80967/googletest/CMakeLists.txt#L47

and ggooglemock/CMakeLists.txt:  project(gmock VERSION 1.9.0 LANGUAGES CXX C) - https://github.com/google/googletest/blob/b88511ef640c046ec9c7553b383a2aba64c80967/googlemock/CMakeLists.txt#L40

Reply:
@gennadiycivil That's apparently a `cleanup-cmake` branch, not this branch.

Reply:
@dakotahawkins Sorry , we have a disconnect. 
Just look in the master brach, https://github.com/google/googletest/blob/master/googlemock/CMakeLists.txt line 40. 
Thanks
G

Reply:
Those have been replaced in this commit with `${GOOGLETEST_VERSION}`. See [here](https://github.com/google/googletest/pull/1682/files#diff-98eb25640483d40b7d2663b41473906aR40) and [here](https://github.com/google/googletest/pull/1682/files#diff-7b12e50666f60ab6d4721a97c7b54868R47).

Reply:
Thank you